lib/ims/lti/services/message_authenticator.rb

```diff
@@ -14,7 +14,7 @@ def initialize(launch_url, params, secret)
 
 
     def valid_signature?
-      simple_oauth_header.valid?(signature: signature)
+       message.jwt ? valid_jwt? : simple_oauth_header.valid?(signature: signature)
```

Comment:
At some point it would probably be good to have two separate classes to that we don't have to have a bunch of if logic in here to determine whether we are dealing with jwt or oauth